lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -221,7 +221,8 @@ static void emitZeroWidthIndexingValue(PPS &os) {
 
 /// Return the verilog name of the port for the module.
 static StringRef getPortVerilogName(Operation *module, size_t portArgNum) {
-  return cast<HWModuleLike>(module).getPort(portArgNum).getVerilogName();
+  auto hml = cast<HWModuleLike>(module);
+  return hml.getPort(portArgNum).getVerilogName();
```

Comment:
Nit: unrelated change? Also, IMO inlining single-use seems better.

Reply:
Unrelated, but matches other code adjacent.  Will be rewritten soon.